client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -25,12 +25,40 @@ registerBlockType( 'jetpack/simple-payments', {
 
 	category: 'jetpack',
 
-	keywords: [ 'simple payments', 'PayPal' ],
+	keywords: [ __( 'simple payments', 'jetpack' ), __( 'PayPal', 'jetpack' ) ],
 
 	attributes: {
+		// @TODO move everything else except `paymentId` to state instead?
+		currency: {
+			type: 'string',
+			default: 'USD',
```

Comment:
FWIW, any block instances that use the default values are removed from the serialized representation of the block, so we should either be sure that we're using the right default values, or try to rely as little as possible on default attribute values at block registration level.